vuepress/docs/next/tutorials/curate/bundle-private-images.md

```diff
@@ -0,0 +1,85 @@
+---
+sidebarDepth: 2
+---
+
+# Install Bundle from a Private Image Registry
+
+This tutorial provides to way to utilize bundles from a private image registry in your Entando projects. The steps below use environment variables to pass the Secret for authentication required by private registries.
+
+**New or Existing Bundle**
+* If you have an existing bundle in a private image registry, start at [Step 1: Create the Registry Credentials](#step-1-create-the-registry-credentials). 
```

Comment:
I don't understand lines 10 and 11. Either way you have to do Step 1. What's the distinction?